src/samVisualize/graphGeneration/cfnTemplateGraphGenerator.ts

```diff
@@ -0,0 +1,152 @@
+/*!
```

Comment:
How is this different from https://github.com/aws/aws-toolkit-vscode/pull/1873? Applies to `graph.ts` as well, and maybe others I've missed.

Reply:
Yes, I believe it's because I used the same branch to open another PR? So it just re-opened the first one? I think I needed to use a different branch for each PR, so that we don't see previous, approved commits.

Reply:
Yes, the PR should be based off of the latest approved changes so that these don't show up.

Reply:
For sure, I'll leave this one open as is and then ensure a new branch for subsequent PRs? That's my mistake

Reply:
Yeah just leave it open since it doesn't seem like there's too much new code in this PR.